Cargo.toml

```diff
@@ -1236,7 +1247,7 @@ setup = [
     "curl",
     "-o",
     "assets/models/bunny.meshlet_mesh",
-    "https://raw.githubusercontent.com/JMS55/bevy_meshlet_asset/defbd9b32072624d40d57de7d345c66a9edf5d0b/bunny.meshlet_mesh",
```

Comment:
I think this file got clobbered in a merge.

Reply:
reverted to the ref on `main`